lxd/container_lxc.go

```diff
@@ -1728,7 +1728,7 @@ func (c *containerLXC) initLXC(config bool) error {
 	}
 
 	// Setup shmounts
-	if lxc.HasApiExtension("mount_injection") || util.RuntimeLiblxcVersionAtLeast(3, 1, 0) {
+	if lxc.HasApiExtension("mount_injection") && util.RuntimeLiblxcVersionAtLeast(3, 1, 0) {
```

Comment:
Similar to previous comment, just checking for the API Extension should be fine right?

Reply:
Also, that change should have been in the previous commit